render.js

```diff
@@ -0,0 +1,24 @@
+const Renderer = function () {
+
+    const addPost = function (post) {
+        $('#posts').append(`<div class="post" data-id="${post.id}"></div`)
```

Comment:
You should declare a variable and assign this line to it. 
Then, in the next lines, you can append the rest of the elements to it (instead of selecting the specific element by data-id each time)